src/content/en/fundamentals/security/credential-management/index.md

```diff
@@ -37,65 +29,45 @@ and take a look at the
 
 <div class="clearfix"></div>
 
+### Sign in user
 
-## Steps to implement Credential Management
-
-While there are many ways to successfully integrate the Credential Management
-API, and the specifics of an integration depend on the structure and user
-experience of the site, sites that use this flow have these user experience
-advantages:
-
-* Existing users of your service that have a single credential saved to the
-  browser are immediately signed in, and they are redirected to the signed-in
-  page as soon as authentication finishes.
-* Users that have multiple credentials saved or that have disabled automatic
-  sign-in need to respond to one dialog before they go to the website's
-  signed-in page.
-* When users sign out, the website ensures they are not automatically
-  signed back in.
-
-Key Point: Using the Credential Management API requires the page be served
-from a secure origin.
-
-### Retrieve user credentials and sign in
-
-To sign the user in, retrieve the credentials from the browser's password
-manager and use those to log the user in.
+To sign in the user, retrieve the credentials from the browser's password
+manager and use them to log in the user.
 
 For example:
 
 1. When a user lands on your site and they are not signed in, 
-   call `navigator.credential.get()`
-2. Use the retrieved credentials to sign the user in.
+   call [`navigator.credential.get()`](/web/fundamentals/security/credential-management/reference#navigatorcredentialsget).
+2. Use the retrieved credentials to sign in the user.
 3. Update the UI to indicate the user has been signed in.
 
 Learn more in
-[Retrieve Credentials](/web/fundamentals/security/credential-management/retrieve-credentials).
+[Sign In Users](/web/fundamentals/security/credential-management/retrieve-credentials#auto-sign-in).
 
 ### Save or update user credentials
 
-If the user signed in with a username and password:
+If the user signed in with a federated identity provider such as Google
+Sign-In, Facebook, GitHub, etc:
 
-1. After the user successfully signs in, creates an account or changes a
-   password, create the `PasswordCredential` with the user ID and
-   the password.
-2. Save the credential object using `navigator.credentials.store()`.
+1. After the user successfully signs in or creates an account, create the [`FederatedCredential`](/web/fundamentals/security/credential-management/reference#federatedcredential) with the user's email address as
+   the ID and specify the identity provider with `.provider`.
```

Comment:
Good question. According to the API docs in MDN, the official object is FederatedCredential, so it would be FederatedCredential.provider. However, the sample code throughout the docs refer to the object as federated (so federated.provider).

We'd want to make sure we change it everywhere.

@agektmr - what's the best thing to do here?